deploy/docs/Best_Practices.md

```diff
@@ -1030,6 +1031,90 @@ metadata:
         value: '12345'
 ```
 
+## Using newer Kube Prometheus Stack
+
+Due to breaking changes, we do not support the latest Kube Prometheus Stack.
+We are aware that it can be a major issue, so this section describes how to install Kube Prometheus Stack in version `34.10.0`
+to work with our collection.
+
+1. Prepare [values-prometheus.yaml] with Kube Prometheus Stack configuration:
+
+   - copy content of `kube-prometheus-stack` from [values.yaml]
+   - add the following configuration:
+
+     ```yaml
+     prometheus:
+       prometheusSpec:
+         initContainers:
+           - name: "init-config-reloader"
+             env:
+               - name: FLUENTD_METRICS_SVC
+                 valueFrom:
+                   configMapKeyRef:
+                     name: sumologic-configmap
+                     key: fluentdMetrics
+               - name: NAMESPACE
+                 valueFrom:
+                   configMapKeyRef:
+                     name: sumologic-configmap
+                     key: fluentdNamespace
+     ```
+  
+   - remove the following configuration:
+
+     ```yaml
+     prometheus:
+       prometheusSpec:
+         thanos:
+           ## Use our own image until Thanos upstream adds arm64 docker images
+           ## to their registry.
+           baseImage: public.ecr.aws/sumologic/thanos
+           version: v0.23.1
+     kube-state-metrics:
+       ## Use the GCR repo, it's more recent and has ARM images starting from 1.9.8
+       image:
+         repository: k8s.gcr.io/kube-state-metrics/kube-state-metrics
+         tag: v1.9.8
+     prometheus-node-exporter:
+       image:
+         tag: v1.3.1
+     ```
+
+1. [Upgrade Kube Prometheus Stack][kube-prometheus-stack]
+
+   ```bash
+   helm repo add prometheus-community https://prometheus-community.github.io/helm-charts
+   helm repo update
+
+   helm install kube-prometheus prometheus-community/kube-prometheus-stack \
+     --namespace sumologic \
+     --create-namespace \
```

Comment:
I tested installation of kube-prometheus-stack only in the same namespace in which sumologic helm chart is installed. I propose to remove --create-namespace and replace command to this form:

```
helm install kube-prometheus prometheus-community/kube-prometheus-stack \
  --namespace <NAMESPACE> \
  --version 34.10.0 \
  -f values.yaml
```

with notice information that namespace should be set to the namespace in which sumologic helm chart is installed.